chain/client/src/sync/state.rs

```diff
@@ -136,11 +134,11 @@ pub struct StateSync {
 
     /// When the "sync block" was requested.
     /// The "sync block" is the last block of the previous epoch, i.e. `prev_hash` of the `sync_hash` block.
-    last_time_block_requested: Option<DateTime<Utc>>,
+    pub last_time_block_requested: Option<DateTime<Utc>>,
```

Comment:
These two need to move to either Client Actor or the client object. 
I am planning to move it when I refactor this file.

Reply:
It feels a bit sad to me to break with the style here and make these two fields pub when none of the other ones are. Before this PR, the only real way to interact with this struct is via pub functions. Why not just add some pub functions to set and get this value and the timeout value below? Or if you like, notice that after this PR, this field is only ever touched in the client actor, which could imply that it's better to just move that field to belong to the client actor